src/typing_manager.ts

```diff
@@ -52,6 +52,21 @@ export class TypingManager implements Disposable {
         this.disposables.push(commands.registerCommand("vscode-neovim.send", this.onSendCommand));
         this.disposables.push(commands.registerCommand("vscode-neovim.send-blocking", this.onSendBlockingCommand));
         this.disposables.push(commands.registerCommand("vscode-neovim.escape", this.onEscapeKeyCommand));
+        this.disposables.push(
+            commands.registerCommand("vscode-neovim.toggle", () => {
```

Comment:
can you factor this out to a `onToggleCommand` function

Reply:
done